client/client.go

```diff
@@ -19,7 +19,7 @@ type Client struct {
 }
 
 func Make(token string, baseUrl string, debug bool, key, cert string) (*Client, error) {
-	restyClient := resty.New().SetDebug(debug).SetHeader("Authorization", "Bearer "+token)
+	restyClient := resty.New().SetDebug(debug).SetHeader("Authorization", "Bearer "+token).SetHeader("CDK-CLIENT", "CLI")
```

Comment:
can we add the version of the cli used ? like `CLI/0.1.1`
Also instead of custom header, we could re-use UserAgent http header

Reply:
For now supported values are "CLI" and "Frontend". It will be used to compute statistics on the public API and should be easy to group.
We will also set the same header on the frontend where you don't want to change the UserAgent.

However, I agree that we could set the UserAgent to identify the CLI and which version it's running.